networker/networker.go

```diff
@@ -1505,6 +1513,54 @@ func (netw *Combined) allowFileshare(publicKey string, address netip.Addr) error
 	return nil
 }
 
+func (netw *Combined) undenyDNS() error {
+	ruleName := "deny-private-dns"
+
+	ruleIndex := slices.Index(netw.rules, ruleName)
+
+	if ruleIndex == -1 {
+		return nil
+	}
+
+	if err := netw.fw.Delete([]string{ruleName}); err != nil {
+		return err
+	}
+	netw.rules = slices.Delete(netw.rules, ruleIndex, ruleIndex+1)
+
+	return nil
+}
+
+func (netw *Combined) denyDNS() error {
+	ruleName := "deny-private-dns"
+	rules := []firewall.Rule{{
+		Name:           ruleName,
+		Direction:      firewall.Outbound,
+		Protocols:      []string{"udp", "tcp"},
+		Ports:          []int{53},
+		PortsDirection: firewall.Destination,
+		RemoteNetworks: []netip.Prefix{
+			netip.MustParsePrefix("10.0.0.0/8"),
```

Comment:
[for future] we should define those ranges in one place, because they are redefined in several places.